avalon/tools/cbloader/widgets.py

```diff
@@ -332,3 +337,57 @@ def __init__(self, parent=None):
 
     def set_version(self, version_id):
         self.data.set_version(version_id)
+
+
+class FilterWidget(QtWidgets.QGroupBox):
+
+    def __init__(self, parent=None):
+        super(FilterWidget, self).__init__(parent=parent)
+
+        self.setTitle("Families")
+
+        MULTI_SELECT = QtWidgets.QAbstractItemView.ExtendedSelection
+
+        layout = QtWidgets.QVBoxLayout()
+
+        checkbox_list = QtWidgets.QListWidget()
+        checkbox_list.setSelectionMode(MULTI_SELECT)
+        checkbox_list.setAlternatingRowColors(True)
+
+        layout.addWidget(checkbox_list)
+
+        self.checkbox_list = checkbox_list
+        self.mainlayout = layout
+
+        self.setLayout(layout)
+
+    def create_filters(self):
```

Comment:
Would you ever call this on its own? :) Not sure why it needs the separate `refresh()` from this? Can we simplify this?

Should we refactor this method to `refresh()`?

Reply:
Fixed in latest commit